assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,48 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+    let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 
+    let anotherFunction = (param) => {
+        console.log(param);
+        return param;
+    };
+
+    anotherFunction(2);
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+    let add = (param1, param2) => {
+        console.log(param1 + param2);
+        return param1 + param2;
+    };
+    add(1,2);
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
 
+    let subtract = (param1, param2) => {
+        console.log(param1 - param2);
+        return param1 - param2;
+    };
+    subtract(1,2);
+
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+    exampleArray = [1,2,3,4];
+    const triple = exampleArray.map((num) => {
+        return num * 3;
+    });
```

Comment:
Since you only have a single statement/expression that you're returning in your `map` callback, you could shorten it (by loosing the braces and the `return` keyword):
```javascript
    const triple = exampleArray.map((num) => num * 3);
```

You can also lose the parentheses if you only have a single parameter (and you're not doing anything fancy with it):
```javascript
    const triple = exampleArray.map(num => num * 3);
```